src/index.html

```diff
@@ -665,34 +665,102 @@ <h3 class="padding-bottom-large">FAQs</h3>
+                                            <a href="#collapse21" class="accordion-title"><h5>Why are some variants in a different strand than the one dbSNP reports?</h5></a>
+                                            <div class="accordion-content" data-tab-content>
+                                                <p>
+                                                    The EVA accepts variant submission in the forward strand only, whereas dbSNP is not so restrictive. In addition to this, dbSNP registers multiple orientations:
+                                                </p>
+                                                <p>
+                                                    <ul>
+                                                        <ul>
+                                                            <li>Contig to chromosome</li>
+                                                            <li>SNP to contig</li>
+                                                            <li>SubSNP (ss) to RefSNP (rs)</li>
+                                                        </ul>
+                                                    </ul>
+                                                </p>
+                                                <p>
+                                                    As a result, variants with a ‘reverse’ orientation will be displayed differently despite the underlying data being equivalent.
+                                                </p>
+                                            </div>
+                                        </li>
+                                        <li class="accordion-item" data-accordion-item>
+                                            <a href="#collapse22" class="accordion-title"><h5>Why can't I find some of the dbSNP accessions on the EVA website?</h5></a>
+                                            <div class="accordion-content" data-tab-content>
+                                                <p>
+                                                    We will gradually import the dbSNP data, and your species of interest may not have been yet processed. During October 2017, we will add to the website a status report on the transition process.
+                                                </p>
+                                                <p>
+                                                    If a species has been already imported into the EVA, the most probable reason is that the dbSNP variant did not satisfy the EVA submission requirements. Please click <a href="?Submit%20Data">here</a> to learn more about the EVA submission requirements.
+                                                </p>
+                                                <p>
+                                                    If you know the variant satisfied all of them, please communicate the issue via <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a>

```

Comment:
Let's rephrase this a bit: `If you know a variant satisfied all of them and it is not displayed in our browser, please`...